### DISCUSSION TITLE
BT-625 Units

### DISCUSSION THREAD

**Discussion by AnnaKSZ** [OP-TED/eForms-SDK#647] [Q&A] [answered]
Hello Lab,

There is the BT-625 with [measurement-unit ](https://docs.ted.europa.eu/eforms/latest/reference/code-lists/measurement-unit.html ) codelist.
There is no piece or similar unit in the codelist.

How can we define the measurement if we want to buy cars or containers or tables .. ?

Thanks in advance
Anna

**Comment by YvesJo** [comment, marked as answer]
Hi,
There is indeed no "unit" or "piece" equivalent available in the "measurement-unit" codelist. Following the rules, the "unitCode" attribute is optional while it is expected by the schema. As the element may not be empty and no suitable code exist, I raise your conversation as an issue.

**Comment by AnnaKSZ** [comment]
Thanks for your response
Anna
